samples/wifi/sta/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _wifi_station_sample:
+
+Wi-Fi: Station
+##############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Station sample demonstrates how to connect the Wi-Fi station to a specified access point.
```

Comment:
Would add `using DHCP` or similar, since there seems to be no static IP fallback for this example